src/mame/jaleco/megasys1.h

```diff
@@ -157,20 +154,19 @@ class megasys1_state : public driver_device
 	int m_mcu_hs = 0;
 	u16 m_mcu_hs_ram[0x10]{};
 
+	const uint16_t* m_gatearray_seq = nullptr;
```

Comment:
Please put the right-associative unary `*` qualifier to the right of the space.

Reply:
I keep forgetting this because it's at odds with standards everywhere else I do work, and at odds with visual studio's default auto reformatting, but fixed now.